deploy/osd-machine-api/010-machine-api.srep-infra-healthcheck.MachineHealthCheck.yaml

```
@@ -14,9 +14,9 @@ spec:
       operator: Exists
   unhealthyConditions:
   - type:    "Ready"
-    timeout: "480s"
+    timeout: "600s"
```

```suggestion
    timeout: "480s"
```
We can keep this at 480s - this controls how fast the MHC controller effectively notices that a node is explicitly NotReady `type: Ready, status: False` and considers it for replacement.

The second one is essentially a timeout how long a machine exists, but is not a node yet.